xml/installation-installation-ironic-ironic_provisioning.xml

```diff
@@ -154,6 +227,77 @@ Deploy_ramdisk : openstack-ironic-image.x86_64-8.0.0.iso</screen>
    <replaceable>remove</replaceable></literal> will remove a property.
   </para>
  </section>
+ <section xml:id="register-redfish-node">
+  <title>Registering a Node with the Redfish Driver</title>
+  <para>
+   Nodes configured to use the Redfish driver should have the driver property
+   set to <literal>redfish</literal>.
+  </para>
+  <para>
+   The following properties are specified in the <literal>driver_info</literal>
+   field of the node:
+  </para>
+  <variablelist>
+   <varlistentry>
+    <term>redfish_address (required)</term>
+    <listitem>
+     <para>
+      The URL address to the Redfish controller. It must include the authority
+      portion of the URL, and can optionally include the scheme. If the scheme
+      is missing, HTTPS is assumed.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>redfish_system_id (required)</term>
+    <listitem>
+     <para>
+      The canonical path to the System resource that the driver will interact
+      with. It should include the root service, version and the unique resource
+      path to the System. For example,<literal> /redfish/v1/Systems/1</literal>.
```

Comment:
the system*

Reply:
I see why it's uppercase, just seems a bit weird to put in the sentence too. Thoughts?